libnopegl/src/backends/gl/rendertarget_gl.c

```diff
@@ -76,13 +76,9 @@ static void resolve_draw_buffers(struct rendertarget *s)
         if (i == 0)
             flags |= GL_DEPTH_BUFFER_BIT | GL_STENCIL_BUFFER_BIT;
         ngli_glReadBuffer(gl, GL_COLOR_ATTACHMENT0 + (GLenum)i);
-#if defined(TARGET_DARWIN)
-        ngli_glDrawBuffer(gl, GL_COLOR_ATTACHMENT0 + (GLenum)i);
-#else
```

Comment:
It might make sense to mention the issue (by referencing b56f9510032d07d989f0875c4f755ce6412eb1fe in the commit description)

Reply:
Done.